### DISCUSSION TITLE
how to log in to minecraft without using the browser

### DISCUSSION THREAD

**Discussion by 59iq** [ORelio/Minecraft-Console-Client#1812] [Q&A]
ok so the proble is that i made it so i can log in the mc copying a browser and puting it in to the client but thats a problem becos takes a while and i cant conect  multiple acc so what can be done about this
![894](https://user-images.githubusercontent.com/93533743/139688997-12369ae6-025e-4053-a6c6-2dbf26e9e09f.PNG)
?

**Comment by ReinforceZwei** [comment]
Use `method=mcc`.
